packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/explorer/services/ExplorerChildrenProvider.java

```diff
@@ -117,8 +123,14 @@ private List<Object> getDefaultChildren(VariableManager variableManager) {
                     representationMetadata.sort(Comparator.comparing(RepresentationMetadata::getLabel));
                     result.addAll(representationMetadata);
                     List<Object> contents = this.objectService.getContents(self);
+                    if (self instanceof Entity entity) {
+                        result.add(((InternalEObject) entity).eSetting(entity.eClass().getEStructuralFeature("superTypes")));
+                    }
                     result.addAll(contents);
+                } else if (self instanceof Setting setting) {
+                    result.addAll((Collection<? extends Object>) setting.get(true));
```

Comment:
If we have an entity then the super types setting is a child and it needs to be considered here, I'll update the PR to take it into account